src/olympia/constants/base.py

```diff
@@ -431,4 +431,4 @@
     '@mozilla.com', '@mozilla.org')
 
 TAAR_ALLOWED_PARAMETERS = (
```

Comment:
Might be more accurate to rename this to `DISCO_API_ALLOWED_PARAMS` or similar at some point.

Reply:
https://github.com/mozilla/addons-server/pull/7493/commits/26cbb2bea072ab0d7b14e558e976a957b9829876